src/test/java/io/confluent/connect/jdbc/JdbcSourceTaskUpdateTest.java

```diff
@@ -418,16 +441,19 @@ public void testCustomQueryWithTimestamp() throws Exception {
   }
 
   private void startTask(String timestampColumn, String incrementingColumn, String query) {
-    startTask(timestampColumn, incrementingColumn, query, 0L);
+    startTask(timestampColumn, incrementingColumn, null, query, 0L);
   }
 
-  private void startTask(String timestampColumn, String incrementingColumn, String query, Long delay) {
+  private void startTask(String timestampColumn, String incrementingColumn, String usePrimaryKeyAsIncrementingColumn, String query, Long delay) {
```

Comment:
Is there a reason this is `String` instead of `boolean` (or `Boolean` to make the `toString()` conversion you might need easier)?